grype/pkg/package_test.go

```diff
@@ -883,8 +883,8 @@ func catalogWithOverlaps(packages []string, overlaps []string) *sbom.SBOM {
 		to := toPkg(parts[1])
 
 		relationships = append(relationships, artifact.Relationship{
-			From: from,
-			To:   to,
+			From: &from,
+			To:   &to,
```

Comment:
Is there a way we could have _both_ pointers and non-pointers? maybe the first iteration add a pointer and subsequent ones add non-pointers?

Reply:
@kzantow I've added your suggestion - how does that look?